src/robot_start_stop.py

```diff
@@ -0,0 +1,45 @@
+from flask import Blueprint, request, jsonify
+import subprocess
+
+blueprint = Blueprint('robot_start_stop', __name__)
+
+@blueprint.route('/start', methods=['POST'])
+def claim_and_start_robot():
+    user = request.form['user']
+
+    result = subprocess.check_call('robot claim -f --username ' + user + ' --email ' + user + ' -m "teleoperating the robot (claimed via Robot Web Server)"', shell=True)
+    if (result != 0):
+        return jsonify({'status': 'error', 'message': 'failed to claim robot'})
+
+    result = subprocess.check_call('robot start -f', shell=True)
```

Comment:
I think when subprocess fails, it actually raises a CalledProcessError [(check_call documentation)](https://docs.python.org/2/library/subprocess.html#subprocess.check_call).